code/obj/item/device/voltron.dm

```diff
@@ -254,7 +254,7 @@
 
 			activating = 1
 
-			playsound(src, "sound/effects/singsuck.ogg", 40, 1)
+			playsound(get_turf(src), 'sound/effects/singsuck.ogg', 40, 1)
```

Comment:
why did you change this to a direct file reference with '.ogg'

Reply:
Ah, I was not sure, which one was correct, and the sound was not working.
I had seen both variants in different places in the codebase, so I figured I would try it like this to see if it fixes it, but it turns out that something else was the problem.
I shall revert it.